features/step_definitions/element_explicit_waiting_steps.rb

```diff
@@ -1,25 +1,16 @@
 # frozen_string_literal: true
 
 When('I wait for the element that takes a while to appear') do
```

Comment:
What is considered "a while"? Also, you are overriding the wait within the code, which hides the behavior from what the actual step implies.

Reply:
this is a legacy step and probably needs some more feng shui, again in a separate PR or combined with the other work.